src/chrome/content/rules/Deutsche-Rentenversicherung.de.xml

```diff
@@ -0,0 +1,11 @@
+<ruleset name="Deutsche-Rentenversicherung.de">
+	<target host="www.deutsche-rentenversicherung.de" />
+	<target host=    "deutsche-rentenversicherung.de" />
+	<target host="www.deutsche-rentenversicherung-bund.de" />
+	<target host=    "deutsche-rentenversicherung-bund.de" />
+	<target host="www.drv-bund.de" />
+	<target host=    "drv-bund.de" />
+
+	<rule from="^https?://(?:www\.)?(?:deutsche-rentenversicherung\.de|deutsche-rentenversicherung-bund\.de|drv-bund\.de)/"
```

Comment:
Building the deterministic finite automaton (DFA) automatically does the optimization, I cannot measure a time difference between the three pattern compilations.